app/backend/src/prisma/prisma.service.ts

```diff
@@ -87,6 +94,36 @@ export class PrismaService extends PrismaClient {
     }
   }
 
+  async userNameExists(username: string): Promise<boolean> {
```

Comment:
@AndrePaulGrenon Delete this function, as you can simply use `getUserIdByNick(nick: string)`, should be renamed to getUserIdByUsername, I'll take care of it